dotfiles/main.tf

```
@@ -39,9 +39,14 @@ variable "coder_parameter_order" {
   default     = null
 }
 
-data "coder_parameter" "dotfiles_uri" {
-  count = var.dotfiles_uri == null ? 1 : 0
+variable "manual_update" {
+  type        = bool
+  description = "Choose whether to enable a Coder app to update dotfiles on demand. This will add a button to workspace page to update dotfiles on demand."
```

Maybe call this refresh or resync instead of update? Update might imply that you're modifying the source dotfiles.

```suggestion
  description = "Choose whether to enable a Coder app to refresh dotfiles on demand. This will add a button to workspace page to refresh dotfiles on demand."
```